google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/Constants.java

```diff
@@ -31,7 +31,7 @@
           .build();
 
   public static final long MAX_PUBLISH_BATCH_COUNT = 1_000;
-  public static final long MAX_PUBLISH_BATCH_BYTES = 3_500_000;
+  public static final long MAX_PUBLISH_BATCH_BYTES = 1024 * 1024 * 7 / 4;  // 3.5 MiB
```

Comment:
I think you want 7/2, not 7/4. This is 1.75 MiB.

Reply:
Good catch, fixed.